scripts/Thaumcraft-04-Artifice.zs

```diff
@@ -277,9 +277,9 @@ mods.thaumcraft.Research.addArcanePage("BASICARTIFACE", <Thaumcraft:ItemBaubleBl
 
 // --- Mirrored Glass
 mods.thaumcraft.Arcane.addShaped("BASICARTIFACE", <Thaumcraft:ItemResource:10>, "aer 30, terra 30, ignis 30,", [
-[<ore:bucketEnder>, <ore:gemMercury>, <ore:bucketEnder>],
+[<ore:gemMercury>, <gregtech:gt.blockgem1:8>, <ore:gemMercury>],
```

Comment:
A glass recipe has better a fluid and not a block. why not add cells even if they consumed. Maybe we change the amount of glass each crafting process